### PR TITLE
navbar component

### PR DESCRIPTION
**Pull request by khyatibaria**
### Simple Navigation 
Please review it
image

### CODE REVIEW

File: navigation/navigation.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+nav {
```
Comment (Tarandeep-s1ngh) [inline navigation/navigation.css]: Try not to use an element selector, rather give a class to it.

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: variable.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  --padding-4: 1.75rem;
+  --margin-4: 1.75rem;
+  --padding-5: 2rem;
+  --margin-5: 2rem;
+  --fontSize-0: 1rem;
+  --fontSize-1: 1.25rem;
+  --fontSize-2: 1.5rem;
+  --fontSize-3: 1.75rem;
+  --fontSize-4: 2rem;
+  --radius: 0.25rem;
```
Comment (BelideNikhil) [inline variable.css:56]: Don't use rem for border radius as it will change the shape based on pixel density.
Rest all is good!

**Review by BelideNikhil — APPROVED** [review]

File: variable.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  color: inherit;
+}
+
+body {
+  background-color: var(--bg-color);
+  min-height: 100vh;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+}
```
Comment (MaruthiPothuganti) [inline variable.css:23]: These should be moved into reset CSS category

**Review by MaruthiPothuganti — APPROVED** [review]
Recommended change: move a code snippet into reset.css - Code organization.
Everything else looks good.

**Pull request merged**